ui/client/documents/index.js

```diff
@@ -57,7 +59,7 @@ export const ConfidenceBar = withStyles((theme) => ({
 }))(LinearProgress);
 
 const semanticSearchParagraphs = async(query) => {
-  let url = `/api/dojo/paragraphs/search?query=${query}&size=50`;
+  let url = `/api/dojo/paragraphs/search?query=${query}&size=40`;
```

Comment:
Since the page has good enough results count and both semantic search and semantic highlight may take "long" to respond, I have toned this down to 40 results. Especially while we try it the semantic highlighter on the deployed instance.